interfaces/automated-testing/scd/scd.yaml

```diff
@@ -57,4 +76,26 @@ paths:
         '404':
           description: The USS automated testing interface is not activated.
       summary: Retrieve the status of the USS automated testing interface
-      description: Get the status of the USS automated testing interface
\ No newline at end of file
+      description: Get the status of the USS automated testing interface
+
+  /inject/flight:
```

Comment:
The design sketch has `/inject/flight`, but I think this is the point where we want to look more closely at the precise address of this endpoint.  We should be consistent with other endpoints in this API, so this endpoint should be prefixed with `/v1`.  Also, we should use [RESTful style](https://restfulapi.net/resource-naming/) when naming the resources we are reading or writing -- the RESTful resource collection we're writing to here is "flights" (or possibly "test_flights".  Modeling after [the RID interface](https://github.com/interuss/dss/blob/744b7a4733c785066802533b9aa80485fc68db62/interfaces/automated-testing/rid/injection.yaml#L9), we should be able to PUT a particular "flight", so that leads to and endpoint address of `/v1/flights/{flight_id}`.  The other candidate would be POST `/v1/flights`.  The difference is that the client (USS qualifier) creates the flight ID for a PUT and the server (USS) creates the flight ID for a POST.  For technical idempotency reasons (the POST request would need a request_id parameter to be idempotent, so we may as well make that the flight_id via PUT), I recommend the PUT style.

So, I suggest we change the path to `/v1/flights/{flight_id}`.  Note that this will require the addition of a [`parameters` declaration](https://github.com/interuss/dss/blob/744b7a4733c785066802533b9aa80485fc68db62/interfaces/automated-testing/rid/injection.yaml#L10) in the endpoint.

Reply:
Thank you, I have now updated the YAML.